src/Header/Header.less

```diff
@@ -87,6 +83,9 @@
 		.moon-header-title-below,
 		.moon-header-sub-title-below {
 			line-height: 39px;
+			.enyo-locale-non-latin & {
+				line-height: @moon-non-latin-medium-header-line-height;
```

Comment:
Because test letter ''First Panel  ฟิ้  ไั  ஒ  து'' could not display property with 39px;
They are taller than 39px